docs/sql-ref-syntax-dml-insert-into.md

```diff
@@ -41,7 +41,7 @@ INSERT INTO [ TABLE ] table_identifier [ partition_spec ] [ ( column_list ) ]
 * **partition_spec**
 
     An optional parameter that specifies a comma-separated list of key and value pairs
-    for partitions.
+    for partitions. Note that one can use a typed literal (e.g., date'2019-01-02') for a partition column value.
```

Comment:
shall we update the sql doc for ADD/DROP/RENAME PARTITION as well?

Reply:
Sure, done